mcserver/views.py

```diff
@@ -556,7 +582,11 @@ def set_metadata(self, request, pk):
                 "cols": request.GET.get("cb_cols",""),
                 "placement": request.GET.get("cb_placement",""),
             }
-            
+
+        if "sessionName" in request.GET:
+            if "sessionName" not in session.meta:
+                session.meta["sessionName"] = request.GET.get("sessionName", "")
```

Comment:
Should the default session name being "" or the session ID?

Reply:
I set the default to the empty string because using the ID would show it duplicated on the tables, probably causing spacing problems since the UI is not responsive and the UUIDs are very long.

Reply:
One thing I am not sure about is if, in the frontend, should we show the name before the ID (first column), or after?